src/utils/color.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  codGray: '#181818',
```

Comment:
I would call these 'darkest gray, dark gray' etc so it's easier to know the color. And 'accent' and 'accent light' for the accent colours. What do you think?

Reply:
I totally agree because what if we are going to change them in the future but I kind of couldn't think of good names and that's why I did this but now I got an idea which is cool but what about the others? what can we call them?